sigmastate/src/main/scala/sigmastate/utxo/CostTable.scala

```diff
@@ -84,6 +84,7 @@ object CostTable {
   val comparisonPerKbCost = 10
 
   val logicCost = 10
+  val logicBaseCost = 0  // TODO soft-fork: increase it to 20
```

Comment:
Please add description what is this at all

Reply:
added ScalaDoc

Reply:
Seems for 

"("AND", "(Coll[Boolean]) => Boolean", logicBaseCost)" breaking cost change was introduced

Reply:
rolled back all changes in CostTable